docs/create_rules.md

```diff
@@ -2,8 +2,9 @@
 
 In order to lint and score models, `dbt-score` uses a set of rules that are
 applied to each model. A rule can pass or fail when it is run. Based on the
-severity of the rule models are scored. `dbt-score` has a set of rules enabled
-by default, which can be found [here](reference/rules/generic.md).
+severity of the rule, models are scored with the weighted average of the rules
```

Comment:
It's a little bit more complicated than the weighted average, right?

Reply:
There's the rule of "critical -> 0", but other than this it's a weighted average, right? Not sure how to express this in a simple manner for the readme

Reply:
Ok let's keep it like this! :+1: